src/platform_linux/platform.c

```diff
@@ -329,51 +472,6 @@ platform_sort_slow(void               *base,
    return qsort_r(base, nmemb, size, cmpfn, cmparg);
 }
 
-platform_status
-platform_condvar_init(platform_condvar *cv, platform_heap_id heap_id)
-{
```

Comment:
This chunk moved up, to be located alongside their sibling other-sync mechanism functions.